### DISCUSSION TITLE
Module '"@prisma/client"' has no exported member 'ResidentWhereInput'.

### DISCUSSION THREAD

**Discussion by YuDaChao** [prisma/prisma#20482] [Q&A]
import { Resident, ResidentWhereInput } from '@prisma/client';

**Comment by nurul3101** [comment]
Hello @YuDaChao 👋 

Can you please elaborate on the error?
Are you trying to import generated prisma types?

**Reply by YuDaChao** [reply]
Thank you very much, I have solved it. But I have a question, Prisma.ResidentWhereInput does not prompt the ResidentWhereInput property
`
import { Prisma } from '@prisma/client';
const where: Prisma.ResidentWhereInput = {};
`

**Reply by nurul3101** [reply]
Do you mean you are not able to access `Prisma.ResidentWhereInput` type?

**Reply by YuDaChao** [reply]
why?
<img width="764" alt="image" src="https://github.com/prisma/prisma/assets/23699765/00f31c26-3b13-41a3-b977-217975edfff6">

this is right
<img width="340" alt="image" src="https://github.com/prisma/prisma/assets/23699765/9c03d912-3451-486a-82cf-030ba27354ba">

**Reply by nurul3101** [reply]
I seem to have access to `ResidentWhereInput`.
Have you invoked `npx prisma generate` command to generate the types?

<img width="715" alt="Screenshot 2023-08-04 at 1 59 38 AM" src="https://github.com/prisma/prisma/assets/13049676/b82b675c-d897-45a0-ac41-01ed4e33f621">

**Reply by nurul3101** [reply]
Let us know in case you are still running into this issue.

If not, please consider marking the discussion as resolved/closed.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
